assets/js/theme/global/modal.js

```diff
@@ -219,24 +231,33 @@ export class Modal {
     setupFocusableElements(modalType) {
         this.$preModalFocusedEl = $(document.activeElement);
 
-        const $collection = focusableElements[modalType]();
-        $collection.get(0).focus();
+        const { $modalTabbableCollection, modalSelector } = focusableElements[modalType]();
 
-        $('#modal').on('keydown', event => this.onTabbing(event, modalType));
+        this.modalSelector = modalSelector;
+
+        $modalTabbableCollection.get(0).focus();
+
+        $(this.modalSelector).on('keydown', event => this.onTabbing(event, modalType));
```

Comment:
Can we use this.$modal instead of $(this.modalSelector)? If not, it's better to set a init value in class constructor